app/frontend/src/components/designer/settings/FormAccessSettings.vue

```diff
@@ -157,7 +157,7 @@ defineExpose({ idpType, userTypeChanged, updateLoginType });
                 <p class="mt-2 mb-0" :lang="locale">
                   {{ $t('trans.formSettings.referenceGuideA') }}
                   <a
-                    href="https://github.com/bcgov/common-hosted-form-service/wiki/Accessing-forms#Notify-the-idim-team-if-you-are-using-bceid"
+                    href="https://developer.gov.bc.ca/docs/default/component/chefs-techdocs/Capabilities/Form-Management/Accessing-forms/"
```

Comment:
It looks like the anchor was lost here. Do we want this to be
```suggestion
                    href="https://developer.gov.bc.ca/docs/default/component/chefs-techdocs/Capabilities/Form-Management/Accessing-forms/#to-use-log-in-required-with-bceid"
```